crates/re_viewer_context/src/viewer_context.rs

```diff
@@ -26,9 +26,9 @@ pub struct ViewerContext<'a> {
     /// Registry of all known classes of space views.
     pub space_view_class_registry: &'a SpaceViewClassRegistry,
 
-    /// The current recording.
-    /// TODO(jleibs): This can go away
-    pub entity_db: &'a EntityDb,
+    /// The current recording. Short for [`Self.store_context.recording`].
+    /// TODO(jleibs): This can be replaced with `store_context.recording`, perhaps with a helper function.
+    pub recording: &'a EntityDb,
```

Comment:
Given you had to touch every code-location that refers to this anyways, why not just making them `recording()`?

Reply:
I was one commit away from it, but then someone started to review the PR 😆

Reply:
:see_no_evil:

Reply:
I pushed a commit for this now :)